src/app/data-explorer/query-builder-wrapper.service.ts

```diff
@@ -122,6 +194,36 @@ export class QueryBuilderService {
     return of({} as Profile);
   }
 
+  private getQueryBuilderCoreTableProfile(dataModel: DataModel): Observable<Profile> {
+    const requestOptions = {
+      handleGetErrors: false,
+    };
+
+    if (dataModel.id) {
+      return this.profileService.get(
+        CatalogueItemDomainType.DataModel,
+        dataModel.id,
+        'uk.ac.ox.softeng.maurodatamapper.plugins.explorer.querybuilder',
+        'QueryBuilderCoreTableProfileProviderService',
```

Comment:
These strings are used in several places, could we move them into constants for easier maintanence.

Reply:
constants added